bin/test

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+
+require_relative '../utils/diggit'
+
+def verify_version(program, expected, actual)
+  unless expected == actual
```

Comment:
This will only work if the versions are exactly the same (I think; I don't speak Ruby). Do you not want a _minimum_ version?